robot_name: "valkyrie"
path_to_serow_library: "/usr/local/lib"
topics:
  joint_states: "/joint_states"
  imu: "/imu"
  force_torque_states:
    - "/force_torque_states/left"
    - "/force_torque_states/right"
serow_config: "valkyrie.json" 
